repos/fdbt-site/cypress_tests/cypress/support/steps.ts

```diff
@@ -169,12 +184,17 @@ const completePointToPointPeriodProductDetail = (): void => {
     randomlyChooseProductPeriodValidity();
 };
 
-export const completeSinglePages = (csvUpload: boolean, isCarnet: boolean): void => {
+export const completeSinglePages = (csvUpload: boolean, isCarnet: boolean, partialMatching: boolean): void => {
     completeServicePage();
     selectRandomOptionFromDropDown('direction-journey-pattern');
     continueButtonClick();
     completeFareTrianglePages(csvUpload);
-    completeMatchingPage();
+
+    if (partialMatching) {
```

Comment:
As discussed if this doesn't work and we are making it in this ticket then should we remove the changes to the UI tests?